flight_search/README.md

```diff
@@ -12,14 +12,15 @@ The following is a multi-task, multi-agent, multi-tool example for AI-based flig
 Install the required dependencies by running the following command:
 
 ```
-pip install crewai 'crewai[tools]' html2text playwright
+pip install crewai 'crewai[tools]' html2text playwright dotenv
 ```
 
-Set the required environment variables:
+Set the required environment variables in a `.env` file:
 
 ```
 export OPENAI_API_KEY=
 export BROWSERBASE_API_KEY=
+export BROWSERBASE_PROJECT_ID=
```

Comment:
No need for export here, if it's in .env file

Reply:
Good point!